### ISSUE TITLE
the consult for this project

### ISSUE THREAD

**Issue by yaolp123** [zyxbend/TinyUnet#2]
def get_10x_lr_params(self):
     print('run')
        modules = [self]
        for i in range(len(modules)):
            for m in modules[i].named_modules():
                if isinstance(m[1], nn.Conv2d) or isinstance(m[1], SynchronizedBatchNorm2d) \
                        or isinstance(m[1], nn.BatchNorm2d):
                    for p in m[1].parameters():
                        if p.requires_grad:
                            yield p

Hello  when I  run the flowing program: 

x = Variable(torch.randn(1, 3, 360, 480))

model = UNet(3,2,1)

model.get_1x_lr_params()   but there no message merge, why?  and I guess that would show the paramater of the model?
